src/Tasks/Common/Resources/Strings.resx

```diff
@@ -948,5 +948,9 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1213: Targeting .NET 8.0 or higher in Visual Studio 2022 17.7 is not supported.</value>
     <comment>{StrBegin="NETSDK1213: "}</comment>
   </data>
-  <!-- The latest message added is Net8NotCompatibleWithDev177. Please update this value with each PR to catch parallel PRs both adding a new message -->
+  <data name="AssetsFileRuntimeIdentifierMismatch" xml:space="preserve">
+    <value>NETSDK1214: Runtime Identifier {0} is not present in project.assets.json. Please ensure the Runtime Identifier is specified in your project file (e.g., &lt;RuntimeIdentifier&gt;{0}&lt;/RuntimeIdentifier&gt;) and the project was restored before invoking the command.</value>
```

Comment:
This error is very clean, concise, and provides an actionable item. I love it, it's very well done 😄

Reply:
Thank you @nagilson for a thorough review! It's a pleasure developing with you 🙂
I will adress the comments this week.

Reply:
It's a pleasure developing with you too, thank you for your contributions 😄